sentry-android/src/main/java/io/sentry/android/AndroidSentryClientFactory.java

```diff
@@ -73,7 +91,7 @@ public SentryClient createSentryClient(Dsn dsn) {
         }
 
         SentryClient sentryClient = super.createSentryClient(dsn);
-        sentryClient.addBuilderHelper(new AndroidEventBuilderHelper(ctx));
+        sentryClient.addBuilderHelper(new AndroidEventBuilderHelper(this.ctx));
```

Comment:
// nitpick-mode-on
Can you change `this.ctx` to `ctx`?
// nitpick-mode-off

Reply:
sure